astropy/cosmology/funcs/comparison.py

```diff
@@ -36,41 +37,28 @@
 # UTILITIES
 
 
+_CANT_BROADCAST: tuple[type, ...] = (table.Row, table.Table)
```

Comment:
Why does this have an explicit type annotation?

Reply:
Uh oh... Do we have to revert?

Reply:
No. It's just extraneous. If we add a mypy check it will eventually flag it for removal.

Reply:
I would expect any type checker worth using to figure out what the type of `_CANT_BROADCAST` is just from its definition, so it's not clear why we would want to annotate it manually.

Reply:
note that the type that would be inferred by `mypy` is `tuple[type[Row], type[Table]]`, so the annotation is actually *looser*, and that is desirable if `_CANT_BROADCAST` is meant to be re-assigned latter with different elements (is it ?).